features/fork.feature

```diff
@@ -167,10 +170,10 @@ Feature: hub fork
       }
       """
     When I run `hub fork`
-    Then the exit status should be 128
-    And the stderr should contain exactly:
+    Then the exit status should be 0
```

Comment:
Why is the exit status success? Since `mislav` already exists as remote, `hub fork` should fail.

Reply:
This unifies `Unrelated remote already exists` with other related scenarios like `Related fork and related remote already exist`, `Related fork and related remote, but with differing protocol, already exist`, ...

Reply:
I don't see this as a desired change. This PR was supposed to only introduce an extra feature: rename "origin" to "upstream" when attempting to add "origin" fork. Everything else should stay the same.